src/components/modals/AddNewTimeSheet/AddNewTimeSheet.module.css

```diff
@@ -0,0 +1,191 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+.container {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: rgba(0, 0, 0, 0.7);
+}
+
+.modal_time {
+    background-color: #27293d;
```

Comment:
You should use colors from theme.module.css by variable name   var(--dark-background-color) 

theme.module.css  file:
 --dark-background-color: #27293d;

Reply:
I'll get right on to it